src/renderer/inc/IRenderEngine.hpp

```diff
@@ -31,7 +31,7 @@ namespace Microsoft::Console::Render
         std::optional<CursorOptions> cursorInfo;
     };
 
-    class IRenderEngine
+    class __declspec(novtable) IRenderEngine

```

Comment:
Should we have been doing this just always? I literally had no idea this existed, but seems really valuable for interfaces

Reply:
Yeah, we probably should ;P